code/game/objects/item.dm

```diff
@@ -253,27 +253,59 @@
 
 	return ..(user, distance, "", desc_comp)
 
+/obj/item/proc/do_visual_slide(var/turf/from, var/from_pixel_x, var/from_pixel_y, var/turf/target, var/target_pixel_x, var/target_pixel_y)
```

Comment:
I don't know if this proc is more appropriate here or in `code/_helpers/animations.dm`. It seems the only reason it has to be on `/obj/item` to begin with, rather than a more general proc, is that it uses `center_of_mass`?